ResourceStatusSystem/Service/PublisherHandler.py

```diff
@@ -357,42 +357,5 @@ def export_getFreeDiskSpace(self, site, token):
     """ Exporting to web the
     """
 
-    endpoint2Site = {}
-
```

Comment:
you do not need the functionality at all ?

Reply:
This Publisher Web page that would consume these info is broken, and anyway this code would need to be re-done. So, it will stay as it is until the page is restored.